wordpress-mobile/WordPress-iOS
# ULS: Enable new unified flows, disable broken UI tests.

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/404

This enabled the `unifiedWordPress` and `unifiedKeychainLogin` feature flags. It also disables UI tests broken by the new UI. Those will be fixed in a future PR.

To test:
- Verify the UI tests pass on this PR.
- Launch the app.
- If you have saved passwords, verify the iCloud keychain prompt appears.

![keychain](https://user-images.githubusercontent.com/1816888/92285048-976f3280-eec0-11ea-8416-a4ec7cb131b4.jpg)

- Verify the prologue buttons are now `Continue with WordPress.com` and `Enter your site address`.


<img width="400" alt="Screen Shot 2020-09-04 at 11 44 51 AM" src="https://user-images.githubusercontent.com/1816888/92270914-173ad400-eea4-11ea-9ebd-8441c455a9a2.png">

- Select `Continue with WordPress.com`. Verify the Get Started view is displayed.

<kbd>![Simulator Screen Shot - iPhone 11 Pro Max - 2020-09-04 at 11 45 24](https://user-images.githubusercontent.com/1816888/92270988-45201880-eea4-11ea-949c-bc6ab451b7f7.png)</kbd>

- Select `Enter your site address`. Verify the Site Address view is displayed.

<kbd>![Simulator Screen Shot - iPhone 11 Pro Max - 2020-09-04 at 11 45 28](https://user-images.githubusercontent.com/1816888/92270998-494c3600-eea4-11ea-8afd-c17f5ea868a1.png)</kbd>


PR submission checklist:
